src/vaibhav/PersonalBanking.java

```diff
@@ -0,0 +1,191 @@
+package vaibhav;
+
+public class PersonalBanking {
+	
+	int iBal,sBal,hBal,bBal;
+	int iCredit,sCredit,hCredit,bCredit,iDebit,sDebit,hDebit,bDebit;	
+	
+	void bankOperation(String operation,int amount,String bank) {
+		switch(operation) {
+			case "Credit" :
+				creditOperation(bank,amount);
+				break;
+			case "Debit" :
+				debitOperation(bank,amount);
+				break;
+			default :
+				System.out.println("Enter Valid details to proceed");		
+		}		
+	}
+	
+	void creditOperation(String bank,int amount) {
+		switch(bank) {
+			case "ICICI":
+				iBal +=amount;
+				iCredit++;
+				break;
+			case "SBI":
+				sBal +=amount;
+				sCredit++;
+				break;
+			case "HDFC" :
+				hBal +=amount;
+				hCredit++;
+				break;
+			case "BOB" :
+				bBal +=amount;
+				bCredit++;
+				break;		
+		}		
+	}
+	
+	void debitOperation(String bank,int amount) {
+		switch(bank) {
+			case "ICICI":
+				if (amount < iBal) {
+					iBal -= amount;
+					iDebit++;
+				}
+				else 
+					System.out.println("Amount you are trying to withdraw from ICICI bank is greater than the balance,Please check and try again");
+				break;
+			case "SBI":
+				if (amount < sBal) {
+					sBal -= amount;
+					sDebit++;					
+				}
+				else 
+					System.out.println("Amount you are trying to withdraw from SBI bank is greater than the balance,Please check and try again");
+				break;
+			case "HDFC" :
+				if (amount < hBal) {
+					hBal -= amount;
+					hDebit++;
+				}
+				else 
+					System.out.println("Amount you are trying to withdraw from HDFC bank is greater than the balance,Please check and try again");
+				break;
+			case "BOB" :
+				if (amount < bBal) {
+					bBal -= amount;
+					bDebit++;
+				}
+				else 
+					System.out.println("Amount you are trying to withdraw from BOB bank is greater than the balance,Please check and try again");
+				break;	
+			default :
+				System.out.println("Enter Valid details for debit operation");	
+		}		
+	}
+	
+	void displayTotalBalance(String bank) {
+		switch (bank) {
+		case "ICICI":
+			System.out.println("ICICI Total balance is "+iBal);
+			break;
+		case "SBI":
+			System.out.println("SBI Total balance is "+sBal);
+			break;
+		case "HDFC" :
+			System.out.println("HDFC Total balance is "+hBal);
+			break;
+		case "BOB" :
+			System.out.println("BOB Total balance is " +bBal);
+			break;	
+		default :
+			System.out.println("Enter Valid details to get your total balance");		
+		}		
+	}
+	
+	void totalCreditDebit(String bank) {
```

Comment:
I guess Here you need to print totalCredit operation like this -> icic+bob+hdfc+sbi, same for debit card